aten/src/ATen/native/ts_native_functions.yaml

```diff
@@ -28,6 +28,11 @@ full_codegen:
   - cat
   - clamp
   - clamp_min
+  # clone() *should* show up in the IR.
+  # For example, tracing through at::reshape() will insert a clone in cases where the output
+  # needs to be cloned first before constructing a view, when the input strides are incompatible.
+  # This clone needs to be there in order to faithfully re-execute the view later.
+  - clone
```

Comment:
Oh I forgot to mention this in the PR description, but this is interesting.

LTC used to treat `at::clone()` as a no-op, but here I'm adding it directly to the IR. Description for why is above. LTC was able to get away with this before because on the torchscript side it would always convert "view" nodes into an `at::reshape()` call, which would perform the clone on the fly when necessary.

Reply:
I am guessing pt/xla also need to lower clone?

Reply:
I think pt/xla might be able to get away with not doing it (we can probably just confirm by seeing if CI complains eventually though).

Basically, this code is invalid:
```
>>> a = torch.ones(2, 2)
>>> b = torch.permute_copy(a, (1, 0))
>>> c = torch.view_copy(b, (4,))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: view size is not compatible with input tensor's size and stride (at least one dimension spans across two contiguous subspaces). Use .reshape(...) instead.
>>>
```

But this code is valid:

```
>>> a = torch.ones(2, 2)
>>> b = torch.permute_copy(a, (1, 0))
>>> b_clone = b.clone(memory_format=torch.contiguous_format)
>>> c = torch.view_copy(b_clone, (4,))
>>>
```

The main difference is that the `view_copy` kernel in core (and internally, also the `view` kernel), expect certain guarantees around the strides of the input. Users will usually just call `reshape()`, which will take care of this for you by emitting a `clone()` call in the cases where the input has weird strides.

If the torchscript interpreter (running through the LTC TS backend) tries to execute a graph containing the ops in the first code snippet, it will just execute each of the aten kernels corresponding to those ops, and throw the same error message.

When XLA compiles a graph containing ops in the first snippet though.... I'm pretty sure as long as its implementation of `view_copy` can handle any of the necessary inputs, then we don't need to worry about manually inserting a clone beforehand. It'll be good to eventually confirm with tests though.